### ISSUE TITLE
Implement split_at for outer iter

### ISSUE THREAD

**Issue by bluss** [bluss/rust-ndarray#81]

**Comment by vbarrielle** [comment]
I'd like to try and implement this functionality.
Just to be clear, the signature would be somehting like:
``` rust
fn outer_split_at(&self, index: usize) -> (ArrayView<A, D>, ArrayView<A, D>)
```

I'm also wondering if the split is possible along other dimensions, which would call for a `fn axis_split_at(&self, axis: usize, index: usize)` method.

**Comment by bluss** [comment]
That sounds interesting too, but for this issue I was thinking of split at for the outer iter itself. `OuterIter::split_at(usize) -> (OuterIter, OuterIter)`.

**Comment by vbarrielle** [comment]
I see, that's interesting too. Well I'll have a look at that then :)

**Comment by vbarrielle** [comment]
Should be closed as #86 got merged.

**Comment by bluss** [comment]
oh right, thanks.
